src/actions/exchangeActions.js

```diff
@@ -328,29 +370,31 @@ export const checkEnableExchangeAllowanceTransactionsAction = () => {
       },
       [],
     );
-    exchangeAllowances
-      .filter(({ enabled }) => !enabled)
-      .map(({ transactionHash, fromAssetCode, toAssetCode }) => {  // eslint-disable-line
-        const enabledAllowance = allHistory.find(
-          // $FlowFixMe
-          ({ hash, status }) => hash === transactionHash && status === TX_CONFIRMED_STATUS,
-        );
-        if (enabledAllowance) {
-          const fromAssetData = getAssetData(getAssetsAsList(currentAccountAssets), supportedAssets, fromAssetCode);
-          const toAssetData = getAssetData(getAssetsAsList(currentAccountAssets), supportedAssets, toAssetCode);
-          Toast.show({
-            message: t('toast.exchangeEnabled', {
-              fromAssetName: fromAssetData.name,
-              fromAssetSymbol: fromAssetData.symbol,
-              toAssetName: toAssetData.name,
-              toAssetSymbol: toAssetData.symbol,
-            }),
-            emoji: 'handshake',
-            autoClose: true,
-          });
-          dispatch(enableExchangeAllowanceByHashAction(transactionHash));
-        }
-      });
+    Object.keys(exchangeAllowances).map((accountId) =>
+      exchangeAllowances[accountId]
+        .filter(({ enabled }) => !enabled)
+        .map(({ transactionHash, fromAssetCode, toAssetCode }) => {  // eslint-disable-line
```

Comment:
Looks like this should be 'forEach'

```suggestion
        .forEach(({ transactionHash, fromAssetCode, toAssetCode }) => { 
```

Reply:
Old code, didn't bother to fix as it should have been quick fix, but will change, thanks 👍